TowerForge/TowerForge/GameViewController.swift

```diff
@@ -10,8 +10,10 @@ import SpriteKit
 class GameViewController: UIViewController {
     private var gameWorld: GameWorld?
     var gameMode: Mode?
+    var isPaused = false
```

Comment:
I think there is a state enums that might be of use instead of using another boolean